### ISSUE TITLE
Syncing tabs doesn't work with Android app

### ISSUE THREAD

**Issue by mvevitsis** [floccusaddon/floccus#1650]
### Which version of floccus are you using?

5.2.0

### How many bookmarks do you have, roughly?

5 tabs

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Google Drive

### Describe the Bug

If you have a tabs profile (e.g. tabs.xbel) and add it to the android app (e.g. via the export and import feature), and edit it on Android (e.g. share a URL to Floccus Android app and save in your tabs profile) it does not work when you go to desktop and try to pull it down. Instead you get an error, invalid tab. 

Another problem is the android app only has a sync button, not pull down and push up buttons as far as I can tell. 

### Expected Behavior

I was hoping that I could import the tab profile into the android app and then do the following:
1. Share a URL to tab profile on Android 
2. Sync
3. Pull down on desktop and have it opened as a tab

In short, I want to share a URL to the android app and then open it as a tab on my PC. Isn't that how it should work?

### To Reproduce

Import a tabs profile to Android app, add URL, try pulling it down on desktop 

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
Hey @mvevitsis 
Sorry for the long delay.

I will look into this!

**Comment by marcelklehr** [comment]
> Instead you get an error, invalid tab.

I could not reproduce this. Is it possible that you tried this with a file: URL? Firefox for example does not allow opening tabs with file: URLs automatically, which is likely the reason for the error you got. In the latest version file: tabs should simply be ignored without errors.

> Another problem is the android app only has a sync button, not pull down and push up buttons as far as I can tell.

This is true. For simplicity reasons push and pull are hidden in the profile settings on Android. Could you open a feature request for this?

**Comment by mvevitsis** [comment]
I opened a request referencing your comment
